catalog/dags/data_refresh/data_refresh_types.py

```diff
@@ -68,6 +76,16 @@ def __post_init__(self):
         refresh_matview_timeout=timedelta(hours=72),
         create_pop_constants_view_timeout=timedelta(hours=24),
         create_materialized_view_timeout=timedelta(hours=72),
+        data_refresh_poke_interval=int(os.getenv("DATA_REFRESH_POKE_INTERVAL", 60)),
+        filtered_index_poke_interval=int(
+            os.getenv("FILTERED_INDEX_POKE_INTERVAL", 60 * 30)
```

Comment:
Since this is referencing a new environment variable, would you mind adding it to the catalog's `env.template` [here](https://github.com/WordPress/openverse/blob/main/catalog/env.template#L103), similar to the `DATA_REFRESH_POKE_INTERVAL`? Alternatively I actually think it's fine to reuse the `DATA_REFRESH_POKE_INTERVAL` here as well, since we basically just use that to make all the poke intervals very quick in local testing.

Reply:
I will reuse `DATA_REFRESH_POKE_INTERVAL`.